examples/shared_samples/ethernet_network.rb

```diff
@@ -32,6 +40,15 @@
 # api_version = 500 & variant = Synergy to OneviewSDK::API500::Synergy::EthernetNetwork
 # api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::EthernetNetwork
 # api_version = 600 & variant = Synergy to OneviewSDK::API600::Synergy::EthernetNetwork
+# api_version = 800 & variant = Synergy to OneviewSDK::API800::Synergy::EthernetNetwork
+# api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::EthernetNetwork
+# api_version = 1000 & variant = C7000 to OneviewSDK::API1000::C7000::EthernetNetwork
+# api_version = 1000 & variant = Synergy to OneviewSDK::API1000::Synergy::EthernetNetwork
+# api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::EthernetNetwork
+# api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::EthernetNetwork
```

Comment:
I am still not able to see variant Synergy. Please change it. And please have a look in all other files and for other apiversions.